projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/case-flag-table/case-flag-table.component.html

```diff
@@ -2,32 +2,32 @@
   <caption class="govuk-table__caption govuk-table__caption--l" *ngIf="tableCaption">{{tableCaption}}</caption>
   <thead class="govuk-table__head">
     <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header col-party-name">{{firstColumnHeader}}</th>
-      <th scope="col" class="govuk-table__header col-comments">Comments</th>
-      <th scope="col" class="govuk-table__header col-creation-date">Creation date</th>
-      <th scope="col" class="govuk-table__header col-last-modified">Last modified</th>
-      <th scope="col" class="govuk-table__header col-flag-status">Flag status</th>
+      <th scope="col" class="govuk-table__header col-party-name">{{firstColumnHeader | rpxTranslate}}</th>
```

Comment:
@Kartheek-S - no need to translate this value: this is expected to be the party name, or is already translated before being provided as an input value